spec/tree_spec.rb

```diff
@@ -1,14 +1,79 @@
 require 'rspec'
 require 'tree'
 
-describe 'Tree' do
+tree = Tree.new
+
+describe Tree do
   it 'should be a Class' do
-    expect(described_class.is_a? 'Class').to be_true
+    expect(Tree.is_a? Class).to eq true
+  end
+
+  context 'when grows one foot' do
+    it "equals 1" do
+      tree.age!
+      expect(tree.height).to eq(1)
+    end
+  end
+
+  context 'when ages one year' do
+    it "equals 1" do
+      expect(tree.age).to eq(1)
+    end
+  end
+
+  context 'when has no fruit' do
+    it "equals true" do
+      expect(tree.any_apples?).to eq true
+    end
+  end
+
+  context 'when has apples after 3 years' do
+    it "equals true" do
+      tree.age!
+      expect(tree.any_apples?).to be true
+    end
+  end
+
+  context 'when dies after 25 years' do
+    it "equals false" do
+      (1..13).each{ |idx| tree.age!}
```

Comment:
This works. You could also use `.times` which would make this cleaner.
https://ruby-doc.org/core-2.2.2/Integer.html#method-i-times